oca_port/migrate_addon.py

```diff
@@ -25,47 +25,50 @@
-        "\t2) Adapt the module to the {version} version:",
-        f"\t\t=> {bc.BOLD}" "{mig_tasks_url}" f"{bc.END}",
-        (
-            "\t3) On a shell command, type this for uploading the content to GitHub:\n"
-            f"{bc.DIM}"
-            "\t\t$ git add --all\n"
-            '\t\t$ git commit -m "[MIG] {addon}: Migration to {version}"\n'
-            "\t\t$ git push {remote} {mig_branch} --set-upstream"
-            f"{bc.END}"
-        ),
-        "\t4) Create the PR against {from_org}/{repo_name}:",
-        f"\t\t=> {bc.BOLD}" "{new_pr_url}" f"{bc.END}",
-    ]
-)
-BLACKLIST_TIPS = "\n".join(
-    [
-        f"\n{bc.BOLD}{bc.OKCYAN}The next steps are:{bc.END}",
-        (
-            "\t1) On a shell command, type this for uploading the content to GitHub:\n"
-            f"{bc.DIM}"
-            "\t\t$ git push {remote} {mig_branch} --set-upstream"
-            f"{bc.END}"
-        ),
-        "\t2) Create the PR against {from_org}/{repo_name}:",
-        f"\t\t=> {bc.BOLD}" "{new_pr_url}" f"{bc.END}",
-    ]
-)
-SIMPLIFIED_MIG_TIPS = "\n".join(
-    [
-        f"\n{bc.BOLD}{bc.OKCYAN}The next steps are:{bc.END}",
-        ("\t1) Reduce the number of commits " f"('{bc.DIM}OCA Transbot...{bc.END}'):"),
-        f"\t\t=> {bc.BOLD}{MIG_MERGE_COMMITS_URL}{bc.END}",
-        "\t2) Create the PR against {from_org}/{repo_name}:",
-        f"\t\t=> {bc.BOLD}" "{new_pr_url}" f"{bc.END}",
-    ]
-)
+MIG_STEPS = {
+    "reduce_commits": (
+        "Reduce the number of commits "
+        f"('{bc.DIM}OCA Transbot...{bc.END}'):\n"
+        f"\t\t=> {bc.BOLD}{MIG_MERGE_COMMITS_URL}{bc.END}"
+    ),
+    "adapt_module": (
+        "Adapt the module to the {version} version:\n"
+        f"\t\t=> {bc.BOLD}"
+        "{mig_tasks_url}"
+        f"{bc.END}"
+    ),
+    "amend_mig_commit": (
+        "Include your changes in the existing migration commit:\n"
+        f"{bc.DIM}"
+        "\t\t$ git add --all\n"
+        "\t\t$ git commit --amend\n"
+        "\t\t$ git push {remote} {mig_branch} --set-upstream"
+        f"{bc.END}"
+    ),
+    "commands": (
+        "On a shell command, type this for uploading the content to GitHub:\n"
+        f"{bc.DIM}"
+        "\t\t$ git add --all\n"
+        '\t\t$ git commit -m "[MIG] {addon}: Migration to {version}"\n'
+        "\t\t$ git push {remote} {mig_branch} --set-upstream"
+        f"{bc.END}"
+    ),
+    "create_pr": (
+        "Create the PR against {from_org}/{repo_name}:\n"
+        f"\t\t=> {bc.BOLD}"
+        "{new_pr_url}"
+        f"{bc.END}"
+    ),
+    "push_blacklist": (
+        "On a shell command, type this for uploading the content to GitHub:\n"
+        f"{bc.DIM}"
+        "\t\t$ git push {remote} {mig_branch} --set-upstream"
+        f"{bc.END}"
+    ),
+}
+MIG_USUAL_STEPS = ("reduce_commits", "adapt_module", "commands", "create_pr")
+MIG_BLACKLIST_STEPS = ("push_blacklist", "create_pr")
+MIG_ADAPTED_STEPS = ("reduce_commits", "adapt_module", "amend_mig_commit", "create_pr")
```

Comment:
These steps are replacing the `SIMPLIFIED_MIG_TIPS` and here is added the `adapt_module` step (and `amend_mig_commit`) as we cannot say that the job did by `odoo-module-migrator` is enough to migrate the module, it's only a helper to spare time.

Reply:
it's a nice work :+1: